src/app/delegates/[addressOrENSName]/page.tsx

```diff
@@ -16,31 +16,39 @@ import {
   fetchCurrentDelegatees,
   fetchCurrentDelegators,
   fetchDelegate,
-  fetchDelegateStatement,
   fetchVotesForDelegate,
 } from "@/app/delegates/actions";
 import { formatNumber } from "@/lib/tokenUtils";
 import {
   processAddressOrEnsName,
+  resolveENSName,
   resolveENSProfileImage,
 } from "@/app/lib/ENSUtils";
 
+// TODO: -> see if react cache can be used in fetchDelegate
+const getCachedDelegate = cache(
```

Comment:
I will be curious to see how [unstable_cache](https://nextjs.org/docs/app/api-reference/functions/unstable_cache) compares to react cache. 

My only comment is to implement this caching on `fetchDelegate` inside `/app/delegates/actions.ts`. This way all requests to fetchDelegates are automatically cached.

Reply:
yep, good catch, agree.